internal/cmd/ini/initializer.go

```diff
@@ -37,6 +37,11 @@ var androidDevicesPatterns = []string{
 
 var iOSDevicesPatterns = []string{"iPad .*", "iPhone .*"}
 
+var fallbackAndroidVirtualDevices = []vmd.VirtualDevice{
+	{Name: "Android Emulator", OSVersion: []string{"11.0", "10.0"}},
```

Comment:
Android Emulator is not a valid emulator name for 11.0 and 10.0.

Reply:
iirc, `Android Emulator` is not a high usage emulator so I'd either remove it or replace it.

Reply:
Oh you're right, I wrongly check the available version when copying the API result (it expected to end a Android 8.0)
I will let only the `Android GoogleAPI Emulator` for this degraded mode.